manifests/install.pp

```diff
@@ -4,31 +4,29 @@
 
   $modulestream_nodejs = $katello_devel::modulestream_nodejs
 
-  if $modulestream_nodejs != undef {
-    package { 'nodejs':
-      ensure   => $modulestream_nodejs,
-      provider => 'dnfmodule',
-    }
+  package { 'nodejs':
```

Comment:
Looking at this, it always should have had a `before => Package['npm']`. Would you mind adding it while you're at it?